cmd/columnify/columnify.go

```diff
@@ -20,7 +20,12 @@ func printUsage() {
 
 func columnify(c columnifier.Columnifier, files []string) (err error) {
 	defer func() {
-		err = c.Close()
+		err2 := c.Close()
+
+		// Don't overwrite existing errors
```

Comment:
[nits] I think it is better to log the error:

```go
if cerr := c.Close(); cerr != nil {
	if err == nil {
		err = cerr
	} else {
		// Don't overwrite existing errors
		log.Println("Failed to close columnifier:", cerr)
	}
}
```

Reply:
fixed, how about that?